contracts/Crowdsale.sol

```diff
@@ -163,7 +152,7 @@ contract Crowdsale is Haltable {
    * Don't expect to just send in money and get tokens.
    */
   function() payable {
-    throw;
+    require(false);
```

Comment:
We could remove the "payable" modifier and the require() to make an empty function (In my opinion the function should be stay there anyway to signal that this is intentional)

Reply:
Removed "payable" modifier and require() to make empty function as requested.